pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1356,10 +1356,30 @@ func doActivateTail(ctx *domainContext, status *types.DomainStatus,
 			status.Key())
 	}
 	status.Activated = true
+	setupVlans(status.VifList)
 	log.Functionf("doActivateTail(%v) done for %s",
 		status.UUIDandVersion, status.DisplayName)
 }
 
+func setupVlans(vifList []types.VifInfo) {
+	for _, vif := range vifList {
+		vlanInfo := ""
+		if vif.Vlan.IsTrunk == true {
+			vlanInfo = fmt.Sprintf("%d-%d", vif.Vlan.Start, vif.Vlan.End)
+		} else if vif.Vlan.Start != 0 {
+			vlanInfo = fmt.Sprintf("%d pvid untagged", vif.Vlan.Start)
+		}
```

Comment:
Don't you need an else case with a continue or something to not set of anything with vlans?

Reply:
I see below there is really no such third case. Either trunk or access. Is that correct?

Reply:
DOne